components/breadcrumb/stories/template.js

```diff
@@ -21,7 +21,7 @@ export const Template = (
 		<ul
 			class=${classMap({
 				[rootClass]: true,
-				[`${rootClass}--${variant}`]: typeof variant !== "undefined",
+				[`${rootClass}--${variant}`]: typeof variant !== "undefined" || variant !== "default",
```

Comment:
This change isn't necessary if we keep variant to an "opt-in" instead of hardcoding a default setting:

```suggestion
				[`${rootClass}--${variant}`]: typeof variant !== "undefined",
```